src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -60,53 +59,43 @@ public static <T> Logger getLogger(Class<T> clazz) {
     }
 
     /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
+     * Creates and returns a base logger with the default configuration and containing a {@code ConsoleHandler}
+     * and a {@code FileHandler}.
```

Comment:
```suggestion
     * Creates and returns a logger named `ab3`, containing a {@code ConsoleHandler}
     * and a {@code FileHandler}.
```

Reason: This is the only thing the method does. It has no control over how the created logger will be used.